e2e/tests/notebook/notebook-owner-actions.spec.ts

```diff
@@ -56,47 +65,39 @@ describe('Workspace OWNER notebook snowman menu actions', () => {
     expect(await analysisPage.isLoaded()).toBe(true);
   });
 
-  test('Duplicate notebook', async () => {
-    await findOrCreateWorkspace(page, { workspaceName: workspace });
+  test('Duplicate rename delete notebook', async () => {
```

Comment:
Combing ``Duplicate notebook` and `Rename notebook` tests together. Avoiding extra login and find. This helps with test runtime deduction.